pkg/apiclient/rollout/rollout.proto

```diff
@@ -71,6 +71,7 @@ message VersionInfo {
 }
 
 message RolloutInfo {
+  k8s.io.apimachinery.pkg.apis.meta.v1.TypeMeta typeMeta = 21;
```

Comment:
nit: I know it looks a bit better to group those fields together at the top of the file but I would prefer to maintain the proto standard and put this new attribute at the bottom of the `RolloutInfo` message (bellow line 98). This would avoid someone duplicating the attribute 21 because he/she didn't pay attention that it was already defined at the top of the message.